src/index.js

```diff
@@ -236,13 +237,16 @@ const MainAppWithApolloProvider = () => {
   }, [globalSettingsState]);
 
   useEffect(() => {
-    !loading && SplashScreen.hideAsync();
-
-    // set orientation to "default", to allow both portrait and landscape
-    ScreenOrientation.lockAsync(ScreenOrientation.OrientationLock.DEFAULT);
+    if(!loading) SplashScreen.hideAsync().then(
+      (result) => {
+        setIsSplashScreenVisible(!result);
+        // set orientation to "default", to allow both portrait and landscape
+        ScreenOrientation.lockAsync(ScreenOrientation.OrientationLock.DEFAULT);
+      }
+    );
```

Comment:
is eslint not complaining here?
i think when using `if` with multiple lines and method chaining, the readability gets heavy. i would prefer:

```
if (!loading) {
  SplashScreen.hideAsync().then(
    (result) => {
      setIsSplashScreenVisible(!result);
      // set orientation to "default", to allow both portrait and landscape
      ScreenOrientation.lockAsync(ScreenOrientation.OrientationLock.DEFAULT);
    }
  );
}
```

or

```
!loading && SplashScreen.hideAsync().then(
  (result) => {
    setIsSplashScreenVisible(!result);
    // set orientation to "default", to allow both portrait and landscape
    ScreenOrientation.lockAsync(ScreenOrientation.OrientationLock.DEFAULT);
  }
);
```

what do you think?

Reply:
why was this change made? would be nice to read about it in the corresponding commit message.

Reply:
changed to the second proposed alternative in e2f9b849f1157cec42932f4fc81cf4ce13f20fc8

I changed it because we unlocked the orientation by setting it to default while the splash screen was still visible.
When I tried it out and changed the orientation to landscape by holding my device horizontally, while the app was still in the splash screen, the orientation changed, while the splash screen kept its format, causing half of the screen to be visible behind the splash screen.

Reply:
nice, thanks